taichi/codegen/llvm/codegen_llvm.cpp

```diff
@@ -355,6 +355,57 @@ TaskCodeGenLLVM::TaskCodeGenLLVM(Kernel *kernel,
 void TaskCodeGenLLVM::visit(DecorationStmt *stmt) {
 }
 
+void TaskCodeGenLLVM::create_value_cast(
```

Comment:
Can `create_value_cast` and `create_fp_trunc` be unified to something like `create_elementwise_unary`?

Reply:
Sure. We can have something like `elementwise_cast` (since we'll need to know the type casting to).